tests/factories/transaction.py

```diff
@@ -73,3 +74,20 @@ class Meta:
     payment_method = SubFactory(InvoiceFactory)
     point_of_sale = SubFactory(PointOfSaleFactory)
     history = SubFactory(HistoryFactory)
+
+
+class TransactionCredit(TransactionFactory):
+    class Meta:
+        model = Transaction
+
+    payment_type = "credit"
+    source = SubFactory(SourceCardPresentFactory)
+
+
+class TransactionBoleto(TransactionFactory):
+    class Meta:
+        model = Transaction
+
+    payment_type = "boleto"
+
+    boleto = SubFactory(InvoiceFactory)
```

Comment:
Esse atributo `boleto` deveria ser `payment_method`.

```suggestion
    payment_method = SubFactory(InvoiceFactory)
```